extension/imos--1.0.sql

```diff
@@ -271,6 +271,28 @@ END;
 $BODY$
 LANGUAGE plpgsql; 
 
+-- Function to add ids to polygons in provided gml
+
+CREATE FUNCTION add_id_to_polygons( p_gml text )
```

Comment:
Well its a bit of a mixed bag in postgis - but I think underscores are winning.  I've switched to that as postgres is case insensitivce when it comes to function naming - so its more readable.